### DISCUSSION TITLE
자기전에 분명히 100%였는데 일어나보니까 방전되어있는 맥북

### DISCUSSION THREAD

**Discussion by mdgarden** [EasyAndBeauty/DailyReceipt-FE-WEB#30] [Q&A]
혹시 다른 분도 겪어보신 적 있으신가요?
제 맥북 이제 1년 좀 넘었는데 벌써 맛이 간걸까요....오늘 일어나서 맥북 안켜져서 넘 놀랐어요 휴ㅠ

**Comment by parkjooyoung0608** [comment]
저는 윈도우지만 혹시 몰라서 항상 자기전에 충전해두고 자요..! 오래쓰니까 확실히 방전 속도가 빨라지더라구요..!

**Comment by IGhost-P** [comment]
저도 가끔 그럴때가 있긴한데 주로 맥북 배터리 문제가 아니라 전기 배선에서 할당 받을수 있는 전력량을 초과하면 그렇더라구요 🥲
